ProductsWebApi/ProductsWebApi/Interface/IUserRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ProductsWebApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProductsWebApi.Interface
+{
+    public interface IUserRepository
+    {
+        Task<bool> FetchUserDetails(string userName, string password);
```

Comment:
Why not as a model object, rather than strings as parameters?

Reply:
changed to model object